annotation-server/test/app.e2e-spec.ts

```diff
@@ -101,10 +116,17 @@ describe('App (e2e)', () => {
             expect(getResponse.body.indication).toEqual('Codeine/indication');
         });
 
+        it('should get details for one guideline', async () => {
+            const getResponse = await request(app.getHttpServer()).get(
+                '/guidelines/' + guidelineId,
+            );
+            expect(getResponse.status).toEqual(200);
+        });
+
         it('should verify guidelines for one medication', async () => {
             const getResponse = await request(app.getHttpServer())
                 .get('/medications/' + codeineId)
-                .query({ withGuidelines: 'true' });
+                .query({ withGuidelines: 'true', getGuidelines: 'true' });
```

Comment:
I guess `withGuidelines` doesn't make sense anymore and is not an accepted query parameter for single medications, right?

Reply:
Addressed in [b18f1dd](https://github.com/hpi-dhc/PharMe/pull/361/commits/b18f1dd7f42836ef7191b9d283bde39e21089f5d)